Rakefile

```diff
@@ -1 +1,95 @@
-require "bundler/gem_tasks"
+require 'bundler/gem_tasks'
+require 'rake/clean'
+require 'cucumber/rake/task'
+require 'mechanize'
+require 'fileutils'
+require 'yaml'
+
+CDK_DOWNLOAD_URL='https://access.redhat.com/downloads/content/293/ver=2/rhel---7/2.0.0/x86_64/product-software'
+CDK_BOX_BASE_NAME='rhel-cdk-kubernetes-7.2-23.x86_64.vagrant'
+
+ADB_DOWNLOAD_URL='http://cloud.centos.org/centos/7/atomic/images'
+ADB_BOX_BASE_NAME='AtomicDeveloperBundle-2.0.0-CentOS7'
+
+CLOBBER.include('pkg')
+CLEAN.include('build')
+
+Cucumber::Rake::Task.new(:features)
+
+namespace :features do
+  cucumber_config = YAML.load_file('./.config/cucumber.yml')
+  cucumber_config.each_key do |key|
+    if key == 'default'
+      next
+    end
+    Cucumber::Rake::Task.new(key.to_sym, "Run Cucumber features using profile '#{key}' (see cucumber.yml)'") do |t|
+      t.profile = key
+    end
+  end
+end
+
+desc 'Download CDK Vagrant box using the specified provider (default \'virtualbox\')'
+task :get_cdk, [:provider] do |t, args|
+  if args[:provider].nil?
+    provider='virtualbox'
+  else
+    provider=args[:provider]
+  end
+  agent = Mechanize.new
+  agent.follow_meta_refresh = true
+  agent.get(CDK_DOWNLOAD_URL) do |page|
+
+    # Submit first form which is the redirect to login page form
+    login_page = page.forms.first.submit
+
+    # Submit the login form
+    after_login = login_page.form_with(:name => 'login_form') do |f|
+     username_field = f.field_with(:id => 'username')
+     username_field.value = 'service-manager@mailinator.com'
+     password_field = f.field_with(:id => 'password')
+     password_field.value = 'service-manager'
+    end.click_button
+
+    # There is one more redirect after successful login
+    download_page = after_login.forms.first.submit
+
+    download_page.links.each do |link|
+     if link.href =~ /#{Regexp.quote(CDK_BOX_BASE_NAME)}-#{Regexp.quote(provider)}.box/
+      download_dir = File.join(File.dirname(__FILE__), 'build', 'boxes')
+      unless File.directory?(download_dir)
+       FileUtils.mkdir_p(download_dir)
+      end
+      agent.pluggable_parser.default = Mechanize::Download
+      puts "Downloading #{link.href}"
+      agent.get(link.href).save(File.join(download_dir, "cdk-#{provider}.box"))
+     end
+    end
+  end
+end
+
+desc 'Download ADB Vagrant box using the specified provider (default \'virtualbox\')'
+task :get_adb, [:provider] do |t, args|
+  if args[:provider].nil?
+    provider='virtualbox'
+  else
+    provider=args[:provider]
+  end
+  agent = Mechanize.new
```

Comment:
Same as this comment https://github.com/projectatomic/vagrant-service-manager/pull/205/files#r62502746